testnets/nibirudevnet3/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "nibirudevnet3",
+  "assets": [
+    {
+      "description": "The native token of Nibiru network",
+      "denom_units": [
+        {
+          "denom": "unibi",
+          "exponent": 0
+        },
+        {
+          "denom": "nibi",
+          "exponent": 6
+        }
+      ],
+      "base": "unibi",
+      "name": "Nibiru",
+      "display": "nibi",
+      "symbol": "NIBI",
+      "logo_URIs": {
```

Comment:
I think logo_URIs is sort of soft-deprecated

Reply:
Should it just be removed?

Reply:
it's fine to keep for now